### DISCUSSION TITLE
May I know what remains not working?

### DISCUSSION THREAD

**Discussion by RosieDu21** [NebulaModTeam/nebula#663] [Q&A] [answered]
Hello, I'm quite glad to see your new released version. I've waited for the new version of your mod since last December,  when DSP updated, and have been seeing you had worked on this for quite a long time. I'm soooo thankful to all the work you did. 

I noticed you mentioned in your readme file that only few things to be done. But I didn't see any relevant PR or task list. Would you have any plan to make one? Or actually those features are so unimportant and little that there's no need to implement them right now? I just want to make sure I can get the most complete experience for you mod.

Wish this doesn't bother you. Thanks a lot.

**Comment by starfi5h** [comment, marked as answer]
You can check that in `List of combat mode syncing features` in README

- Sync kill stats
- Enable Dark Fog communicator (aggressive change and truce)
- Show remote mecha combat drone fleet
- Show remote mecha space craft fleet
- Show ground-to-space attacks animation on client for remote planets (missile turrets, plasma cannon)
- Show space-to-ground attacks animation for remote planets (lancers invading with sweep laser and bomber)
